### PR TITLE
chore: Change components, routes, services, etc to native classes

### PR DESCRIPTION
**Pull request by iamareebjamal**
For #3837

### CODE REVIEW

File: app/routes/application.js [hunk 1]
```diff
@@ -73,15 +79,15 @@ export default Route.extend(ApplicationRouteMixin, {
       eventTypes,
       eventLocations
     };
-  },
+  }
 
   sessionInvalidated() {
     if (!this.session.skipRedirectOnInvalidation) {
-      this._super(...arguments);
+      undefined;
```
Comment (iamareebjamal) [inline app/routes/application.js]: fix

File: app/controllers/public.js [hunk 2]
```diff
@@ -1,24 +1,31 @@
...
+
+  @computed('model.startsAtDate', 'model.endsAtDate')
+  get displayEndDate() {
     return !moment(this.model.startsAtDate).isSame(this.model.endsAtDate, 'minute');
-  }),
-  displaySideMenu: computed('session.currentRouteName', function() {
+  }
+
+  @computed('session.currentRouteName')
+  get displaySideMenu() {
```
Comment (niranjan94) [inline app/controllers/public.js]: Codacy Issue found: [Expected getter 'displaySideMenu' to always return a value.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5274346)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5274346&bid=17273767)

File: app/controllers/public.js [hunk 3]
```diff
@@ -1,24 +1,31 @@
...
 import Controller from '@ember/controller';
 import moment from 'moment';
-import { computed } from '@ember/object';
 
-export default Controller.extend({
-  smallLead: computed('session.currentRouteName', function() {
+@classic
+export default class PublicController extends Controller {
+  @computed('session.currentRouteName')
+  get smallLead() {
```
Comment (niranjan94) [inline app/controllers/public.js]: Codacy Issue found: [Expected getter 'smallLead' to always return a value.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5274346)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5274346&bid=17273767)

File: app/components/tabbed-navigation.js [hunk 4]
```diff
@@ -1,17 +1,22 @@
...
-  classNames : ['tabbed-navigation'],
-  item       : null,
+@classic
+@classNames('tabbed-navigation')
+export default class TabbedNavigation extends Component {
+  item = null;
 
-  currentRoute: computed('session.currentRouteName', 'item', function() {
+  @computed('session.currentRouteName', 'item')
+  get currentRoute() {
```
Comment (niranjan94) [inline app/components/tabbed-navigation.js]: Codacy Issue found: [Expected getter 'currentRoute' to always return a value.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5274346)

File: app/controllers/public.js (hunk 2 above)
Comment (niranjan94) [inline app/controllers/public.js]: Codacy Issue found: [Expected getter 'displaySideMenu' to always return a value.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5274346)

File: app/controllers/public.js (hunk 3 above)
Comment (niranjan94) [inline app/controllers/public.js]: Codacy Issue found: [Expected getter 'smallLead' to always return a value.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5274346)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5274346&bid=17273767)

File: app/controllers/public.js (hunk 2 above)
Comment (niranjan94) [inline app/controllers/public.js]: Codacy Issue found: [Expected getter 'displaySideMenu' to always return a value.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5274346)

File: app/controllers/public.js (hunk 3 above)
Comment (niranjan94) [inline app/controllers/public.js]: Codacy Issue found: [Expected getter 'smallLead' to always return a value.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5274346)

File: app/components/widgets/forms/radio-button.js [hunk 5]
```diff
@@ -1,31 +1,35 @@
+import { tracked } from '@glimmer/tracking';
+import classic from 'ember-classic-decorator';
+import { attributeBindings, tagName } from '@ember-decorators/component';
+import { observes } from '@ember-decorators/object';
+import { computed } from '@ember/object';
```
Comment (niranjan94) [inline app/components/widgets/forms/radio-button.js]: Codacy Issue found: ['computed' is defined but never used.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5274346)

File: app/components/tabbed-navigation.js [hunk 6]
```diff
@@ -1,17 +1,23 @@
...
-  classNames : ['tabbed-navigation'],
-  item       : null,
+@classic
+@classNames('tabbed-navigation')
+export default class TabbedNavigation extends Component {
+  @tracked item = null;
 
-  currentRoute: computed('session.currentRouteName', 'item', function() {
+  @computed('session.currentRouteName', 'item')
+  get currentRoute() {
```
Comment (niranjan94) [inline app/components/tabbed-navigation.js]: Codacy Issue found: [Expected getter 'currentRoute' to always return a value.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5274346)

**Comment by kushthedude** [comment]
Also in several functions, we have misused the computed property at our own will to not return any object, which needs to be fixed before going to classic.

**Comment by iamareebjamal** [comment]
This is an automated conversion. All remaining things have to be converted by hand. Which include correcting things. Goal right now is to fix the build and get it merged

**Pull request merged**